sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcWriteSchemaTransformProvider.java

```diff
@@ -172,71 +312,55 @@ public PCollectionRowTuple expand(PCollectionRowTuple input) {
   @DefaultSchema(AutoValueSchema.class)
   public abstract static class JdbcWriteSchemaTransformConfiguration implements Serializable {
 
-    @Nullable
-    public abstract String getDriverClassName();
```

Comment:
Note that reordering params like this makes this PR significantly harder to review - this would also probably be more helpful as a separate PR if there is a reason to do this kind of reordering; I think the change is fine (though I don't really understand the need), but doing it as part of a large PR with functional changes makes it hard to tell what is cosmetic and what is functional

Reply:
I think all the parameters changed here are just cosmetic, please let me know if that is wrong

Reply:
I also think the same is true of the validate change, again please correct me if that's wrong

Reply:
Purely cosmetic